packages/jbrowse-web/src/plugins/WiggleTrack/model.js

```diff
@@ -0,0 +1,149 @@
+import { types, addDisposer, getRoot, getSnapshot } from 'mobx-state-tree'
+import { autorun } from 'mobx'
+
+import { ConfigurationReference, getConf } from '../../configuration'
+
+import BlockBasedTrack from '../LinearGenomeView/models/blockBasedTrack'
+import WiggleTrackComponent from './components/WiggleTrackComponent'
+import { getParentRenderProps, getContainingView } from '../../util/tracks'
+import { getNiceDomain } from '../WiggleRenderer/util'
+import { checkAbortSignal, isAbortException } from '../../util'
+import WiggleRendererConfigSchema from '../WiggleRenderer/configSchema'
+
+export default (pluginManager, configSchema) =>
+  types.compose(
+    'WiggleTrack',
+    BlockBasedTrack,
+    types
+      .model({
+        type: types.literal('WiggleTrack'),
+        configuration: ConfigurationReference(configSchema),
+        height: types.optional(types.integer, 100),
+      })
+      .actions(self => ({
+        afterAttach() {
+          const getYAxisScale = autorun(
+            function getYAxisScaleAutorun() {
+              const { rpcManager } = getRoot(self)
+
+              const autoscaleType = getConf(self, 'autoscale')
+              const aborter = new AbortController()
+              self.setLoading(aborter)
+
+              if (autoscaleType === 'global') {
+                self.statsPromise = self.adapter.getGlobalStats(aborter.signal)
+              } else if (autoscaleType === 'local') {
+                const { dynamicBlocks, bpPerPx } = getContainingView(self)
+                if (!dynamicBlocks.length) return
+
+                // possibly useful for the rpc group name to be the same group as getFeatures
+                // reason: local stats fetches feature data that might get cached which getFeatures can use
+                self.statsPromise = rpcManager.call(
+                  'statsGathering',
```

Comment:
might want to have the state group be the track's ID, so that each wiggle track's stats calls will go to the same worker, but different tracks can be spread across multiple workers

Reply:
Changed to self.id